apps/desktop/desktop_native/napi/src/lib.rs

```diff
@@ -164,38 +164,42 @@ pub mod sshagent {
     pub struct PrivateKey {
         pub private_key: String,
         pub name: String,
-        pub uuid: String,
+        pub cipher_id: String,
     }
 
     #[napi(object)]
-    pub struct SSHKey {
+    pub struct SshKey {
         pub private_key: String,
         pub public_key: String,
         pub key_algorithm: String,
         pub key_fingerprint: String,
     }
 
     #[napi]
-    pub struct SSHAgentState {
+    pub struct SshAgentState {
         state: desktop_core::ssh_agent::BitwardenDesktopAgent,
     }
 
     #[napi]
-    pub enum SSHKeyImportStatus {
+    pub enum SshKeyImportStatus {
+        // ssh key was parsed correctly and will be returned in the result
         Success,
+        // ssh key was parsed correctly but is encrypted and requires a password
         PasswordRequired,
+        // ssh key was parsed correctly, and a password was provided when calling the import, but it was incorrect
         WrongPassword,
+        // ssh key could not be parsed, either due to an incorrect / unsupported format (pkcs#8) or key type (ecdsa), or because the input is not an ssh key
         ParsingError,
     }
```

Comment:
suggestion: Using `///` will let the IDE pick these up properly
```suggestion
    pub enum SshKeyImportStatus {
        /// ssh key was parsed correctly and will be returned in the result
        Success,
        /// ssh key was parsed correctly but is encrypted and requires a password
        PasswordRequired,
        /// ssh key was parsed correctly, and a password was provided when calling the import, but it was incorrect
        WrongPassword,
        /// ssh key could not be parsed, either due to an incorrect / unsupported format (pkcs#8) or key type (ecdsa), or because the input is not an ssh key
        ParsingError,
    }
```